### ISSUE TITLE
Semaphore CI - timeout

### ISSUE THREAD

**Issue by andykitt** [wix/Detox#1679]
**Describe the bug**
Timeout when running `detox test` in CI pipeline. Test passes locally. 

**Expected behavior**
Detox tests to pass

**Steps to reproduce**
1. Install fresh react native project
2. Setup detox
3. Add `.travis.yml` or `.semaphore.yml` 
4. Run CI pipeline

**Environment (please complete the following information):**
 - Detox: 14.3.4
 - React Native: 0.60.5
 - Node: 
 - Device: iPhone X (12.2)
 - Xcode: 10.2
 - iOS: 12.2
 - macOS: 10.14

**Device and Verbose Detox Logs**
```detox[12084] DEBUG: [exec.js/EXEC_CMD, #1] applesimutils --list --byType "iPhone X" --byOS "12.2"
--
  | detox[12084] DEBUG: [exec.js/EXEC_TRY, #1] Searching for device matching iPhone X...
  | detox[12084] TRACE: [exec.js/EXEC_SUCCESS, #1] [
  | {
  | "availabilityError" : "",
  | "os" : {
  | "bundlePath" : "\/Applications\/Xcode.app\/Contents\/Developer\/Platforms\/iPhoneOS.platform\/Developer\/Library\/CoreSimulator\/Profiles\/Runtimes\/iOS.simruntime",
  | "availabilityError" : "",
  | "buildversion" : "16E226",
  | "availability" : "(available)",
  | "isAvailable" : true,
  | "identifier" : "com.apple.CoreSimulator.SimRuntime.iOS-12-2",
  | "version" : "12.2",
  | "name" : "iOS 12.2"
  | },
  | "availability" : "(available)",
  | "deviceType" : {
  | "name" : "iPhone X",
  | "bundlePath" : "\/Applications\/Xcode.app\/Contents\/Developer\/Platforms\/iPhoneOS.platform\/Developer\/Library\/CoreSimulator\/Profiles\/DeviceTypes\/iPhone X.simdevicetype",
  | "identifier" : "com.apple.CoreSimulator.SimDeviceType.iPhone-X"
  | },
  | "udid" : "08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D",
  | "isAvailable" : true,
  | "state" : "Shutdown",
  | "name" : "iPhone X"
  | }
  | ]
  |  
  | detox[12084] DEBUG: [exec.js/EXEC_CMD, #2] applesimutils --list --byId "08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D"
  | detox[12084] TRACE: [exec.js/EXEC_SUCCESS, #2] [
  | {
  | "availabilityError" : "",
  | "os" : {
  | "bundlePath" : "\/Applications\/Xcode.app\/Contents\/Developer\/Platforms\/iPhoneOS.platform\/Developer\/Library\/CoreSimulator\/Profiles\/Runtimes\/iOS.simruntime",
  | "availabilityError" : "",
  | "buildversion" : "16E226",
  | "availability" : "(available)",
  | "isAvailable" : true,
  | "identifier" : "com.apple.CoreSimulator.SimRuntime.iOS-12-2",
  | "version" : "12.2",
  | "name" : "iOS 12.2"
  | },
  | "availability" : "(available)",
  | "deviceType" : {
  | "name" : "iPhone X",
  | "bundlePath" : "\/Applications\/Xcode.app\/Contents\/Developer\/Platforms\/iPhoneOS.platform\/Developer\/Library\/CoreSimulator\/Profiles\/DeviceTypes\/iPhone X.simdevicetype",
  | "identifier" : "com.apple.CoreSimulator.SimDeviceType.iPhone-X"
  | },
  | "udid" : "08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D",
  | "isAvailable" : true,
  | "state" : "Shutdown",
  | "name" : "iPhone X"
  | }
  | ]
  |  
  | detox[12084] DEBUG: [exec.js/EXEC_CMD, #3] /usr/bin/xcrun simctl boot 08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D
  | detox[12084] DEBUG: [exec.js/EXEC_TRY, #3] Booting device 08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D
  | detox[12084] TRACE: [exec.js/EXEC_SUCCESS, #3]
  | detox[12084] DEBUG: [exec.js/EXEC_CMD, #4] /usr/bin/xcrun simctl bootstatus 08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D
  | detox[12084] TRACE: [exec.js/EXEC_SUCCESS, #4] Monitoring boot status for iPhone X (08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D).
  | [2019-09-26 08:14:45 +0000] Status=2, isTerminal=NO, Elapsed=00:02.
  | Waiting on Data Migration
  | Reason:Gathering plugins
  | Migration Elapsed:00:00 seconds
  |  
  | [2019-09-26 08:14:45 +0000] Status=2, isTerminal=NO, Elapsed=00:02.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.MobileInstallation.-AppMigrator (0AppMigrator.migrator, user-agnostic)
  | Migration Elapsed:00:00 seconds
  |  
  | [2019-09-26 08:14:45 +0000] Status=2, isTerminal=NO, Elapsed=00:03.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.MobileInstallation.SystemAppMigrator (SystemAppMigrator.migrator, user-agnostic)
  | Migration Elapsed:00:00 seconds
  |  
  | [2019-09-26 08:14:46 +0000] Status=2, isTerminal=NO, Elapsed=00:04.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.calendar.migrator (Calendar.migrator)
  | Migration Elapsed:00:01 seconds
  |  
  | [2019-09-26 08:14:47 +0000] Status=2, isTerminal=NO, Elapsed=00:05.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.accountsettings.migrator (AccountMigrator.migrator)
  | Migration Elapsed:00:02 seconds
  |  
  | [2019-09-26 08:14:48 +0000] Status=2, isTerminal=NO, Elapsed=00:06.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.MobileSafari.migrator (MobileSafari.migrator)
  | Migration Elapsed:00:03 seconds
  |  
  | [2019-09-26 08:14:49 +0000] Status=2, isTerminal=NO, Elapsed=00:07.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.MobileSafari.migrator (MobileSafari.migrator)
  | Migration Elapsed:00:04 seconds
  |  
  | [2019-09-26 08:14:50 +0000] Status=2, isTerminal=NO, Elapsed=00:08.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.MobileSafari.migrator (MobileSafari.migrator)
  | Migration Elapsed:00:05 seconds
  |  
  | [2019-09-26 08:14:51 +0000] Status=2, isTerminal=NO, Elapsed=00:09.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.MobileSafari.migrator (MobileSafari.migrator)
  | Migration Elapsed:00:06 seconds
  |  
  | [2019-09-26 08:14:52 +0000] Status=2, isTerminal=NO, Elapsed=00:10.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.MobileSafari.migrator (MobileSafari.migrator)
  | Migration Elapsed:00:07 seconds
  |  
  | [2019-09-26 08:14:53 +0000] Status=2, isTerminal=NO, Elapsed=00:11.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.appleaccount.legacy.migrator (AAAccountLegacyMigrator.migrator)
  | Migration Elapsed:00:08 seconds
  |  
  | [2019-09-26 08:14:54 +0000] Status=2, isTerminal=NO, Elapsed=00:12.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.managedconfiguration.mdm.migrator (MCMDM.migrator)
  | Migration Elapsed:00:09 seconds
  |  
  | [2019-09-26 08:14:55 +0000] Status=2, isTerminal=NO, Elapsed=00:13.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.languageasset.migrator (LanguageAssetMigrator.migrator)
  | Migration Elapsed:00:10 seconds
  |  
  | [2019-09-26 08:14:56 +0000] Status=2, isTerminal=NO, Elapsed=00:14.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.musiclibrary.migrator (MediaLibrary.migrator)
  | Migration Elapsed:00:11 seconds
  |  
  | [2019-09-26 08:14:57 +0000] Status=2, isTerminal=NO, Elapsed=00:15.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.PreferencesMigrator (PreferencesMigrator.migrator)
  | Migration Elapsed:00:12 seconds
  |  
  | [2019-09-26 08:14:58 +0000] Status=2, isTerminal=NO, Elapsed=00:16.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.StocksMigrator (StocksMigrator.migrator)
  | Migration Elapsed:00:13 seconds
  |  
  | [2019-09-26 08:14:59 +0000] Status=2, isTerminal=NO, Elapsed=00:17.
  | Waiting on Data Migration
  | Reason:Running plugin com.apple.iTunesStore.migrator (iTunesStore.migrator)
  | Migration Elapsed:00:14 seconds
  |  
  | [2019-09-26 08:15:01 +0000] Status=4, isTerminal=NO, Elapsed=00:18.
  | Waiting on System App
  |  
  | [2019-09-26 08:15:02 +0000] Status=1, isTerminal=NO, Elapsed=00:19.
  | Waiting on BackBoard
  |  
  | [2019-09-26 08:15:07 +0000] Status=4, isTerminal=NO, Elapsed=00:24.
  | Waiting on System App
  |  
  | [2019-09-26 08:15:08 +0000] Status=4294967295, isTerminal=YES, Elapsed=00:25.
  | Finished
  |  
  |  
  | detox[12084] TRACE: [ArtifactsManager.js/LIFECYCLE] artifactsManager.onBootDevice({ coldBoot: true,
  | deviceId: '08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D' })
  | detox[12084] TRACE: [ArtifactsManager.js/LIFECYCLE] artifactsManager.onBeforeUninstallApp({ deviceId: '08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D',
  | bundleId: 'org.reactjs.native.example.streamline' })
  | detox[12084] DEBUG: [exec.js/EXEC_CMD, #5] /usr/bin/xcrun simctl uninstall 08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D org.reactjs.native.example.streamline
  | detox[12084] DEBUG: [exec.js/EXEC_TRY, #5] Uninstalling org.reactjs.native.example.streamline...
  | detox[12084] TRACE: [exec.js/EXEC_SUCCESS, #5]
  | detox[12084] DEBUG: [exec.js/EXEC_SUCCESS, #5] org.reactjs.native.example.streamline uninstalled
  | detox[12084] DEBUG: [exec.js/EXEC_CMD, #6] /usr/bin/xcrun simctl install 08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D "/Users/semaphore/streamline/ios/build/Build/Products/Debug-iphonesimulator/streamline.app"
  | detox[12084] DEBUG: [exec.js/EXEC_TRY, #6] Installing /Users/semaphore/streamline/ios/build/Build/Products/Debug-iphonesimulator/streamline.app...
  | detox[12084] TRACE: [exec.js/EXEC_SUCCESS, #6]
  | detox[12084] DEBUG: [exec.js/EXEC_SUCCESS, #6] /Users/semaphore/streamline/ios/build/Build/Products/Debug-iphonesimulator/streamline.app installed
  | detox[12084] TRACE: [ArtifactsManager.js/LIFECYCLE] artifactsManager.onBeforeAll()
  | detox[12084] TRACE: [ArtifactsManager.js/LIFECYCLE] artifactsManager.onBeforeTerminateApp({ deviceId: '08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D',
  | bundleId: 'org.reactjs.native.example.streamline' })
  | detox[12084] DEBUG: [exec.js/EXEC_CMD, #7] /usr/bin/xcrun simctl terminate 08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D org.reactjs.native.example.streamline
  | detox[12084] DEBUG: [exec.js/EXEC_TRY, #7] Terminating org.reactjs.native.example.streamline...
  | detox[12084] TRACE: [exec.js/EXEC_SUCCESS, #7]
  | detox[12084] DEBUG: [exec.js/EXEC_SUCCESS, #7] org.reactjs.native.example.streamline terminated
  | detox[12084] TRACE: [ArtifactsManager.js/LIFECYCLE] artifactsManager.onBeforeLaunchApp({ bundleId: 'org.reactjs.native.example.streamline',
  | deviceId: '08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D',
  | launchArgs:
  | { detoxServer: 'ws://localhost:49239',
  | detoxSessionId: 'aad9a32a-e684-bd2c-0680-5cecfdf181c0',
  | detoxPrintBusyIdleResources: 'YES' } })
  | detox[12084] DEBUG: [exec.js/EXEC_CMD, #8] SIMCTL_CHILD_DYLD_INSERT_LIBRARIES="/Users/semaphore/Library/Detox/ios/244486fcee9ff2e760b250ebcbb6568e63bc34d2/Detox.framework/Detox" /usr/bin/xcrun simctl launch 08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D org.reactjs.native.example.streamline --args -detoxServer "ws://localhost:49239" -detoxSessionId "aad9a32a-e684-bd2c-0680-5cecfdf181c0" -detoxPrintBusyIdleResources "YES"
  | detox[12084] DEBUG: [exec.js/EXEC_TRY, #8] Launching org.reactjs.native.example.streamline...
  | detox[12084] TRACE: [exec.js/EXEC_SUCCESS, #8] org.reactjs.native.example.streamline: 12242
  |  
  | detox[12084] DEBUG: [exec.js/EXEC_CMD, #9] /usr/bin/xcrun simctl get_app_container 08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D org.reactjs.native.example.streamline
  | detox[12084] TRACE: [exec.js/EXEC_SUCCESS, #9] /Users/semaphore/Library/Developer/CoreSimulator/Devices/08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D/data/Containers/Bundle/Application/81AEA775-EF2E-412E-B37A-7B71E16344FF/streamline.app
  |  
  | detox[12084] INFO:  [AppleSimUtils.js] org.reactjs.native.example.streamline launched. To watch simulator logs, run:
  | /usr/bin/xcrun simctl spawn 08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D log stream --level debug --style compact --predicate 'processImagePath beginsWith "/Users/semaphore/Library/Developer/CoreSimulator/Devices/08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D/data/Containers/Bundle/Application/81AEA775-EF2E-412E-B37A-7B71E16344FF/streamline.app"'
  | detox[12242] TRACE: [ArtifactsManager.js/LIFECYCLE] artifactsManager.onLaunchApp({ bundleId: 'org.reactjs.native.example.streamline',
  | deviceId: '08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D',
  | launchArgs:
  | { detoxServer: 'ws://localhost:49239',
  | detoxSessionId: 'aad9a32a-e684-bd2c-0680-5cecfdf181c0',
  | detoxPrintBusyIdleResources: 'YES' },
  | pid: 12242 })
  | detox[12084] TRACE: [AsyncWebSocket.js/WEBSOCKET_SEND] {"type":"isReady","params":{},"messageId":-1000}
  | detox[12084] TRACE: [DetoxServer.js/MESSAGE] role=tester action=isReady (sessionId=aad9a32a-e684-bd2c-0680-5cecfdf181c0)
  | detox[12084] DEBUG: [DetoxServer.js/CANNOT_FORWARD] role=testee not connected, cannot fw action (sessionId=aad9a32a-e684-bd2c-0680-5cecfdf181c0)
  | detox[12084] DEBUG: [DetoxServer.js/LOGIN] role=testee, sessionId=aad9a32a-e684-bd2c-0680-5cecfdf181c0
  | detox[12084] DEBUG: [DetoxServer.js/LOGIN_SUCCESS] role=testee, sessionId=aad9a32a-e684-bd2c-0680-5cecfdf181c0
  | detox[12084] TRACE: [DetoxServer.js/MESSAGE] role=testee action=ready (sessionId=aad9a32a-e684-bd2c-0680-5cecfdf181c0)
  | detox[12084] TRACE: [AsyncWebSocket.js/WEBSOCKET_MESSAGE] {"type":"ready","messageId":-1000,"params":{}}
  |  
  | detox[12084] TRACE: [AsyncWebSocket.js/WEBSOCKET_SEND] {"type":"waitForActive","params":{},"messageId":1}
  | detox[12084] TRACE: [DetoxServer.js/MESSAGE] role=tester action=waitForActive (sessionId=aad9a32a-e684-bd2c-0680-5cecfdf181c0)
  | detox[12084] TRACE: [DetoxServer.js/MESSAGE] role=testee action=waitForActiveDone (sessionId=aad9a32a-e684-bd2c-0680-5cecfdf181c0)
  | detox[12084] TRACE: [AsyncWebSocket.js/WEBSOCKET_MESSAGE] {"type":"waitForActiveDone","messageId":1,"params":{}}
  |  
  | Example: should have welcome screen
  | detox[12084] TRACE: [Detox.js/DETOX_BEFORE_EACH] running test: "Example should have welcome screen"
  | detox[12084] TRACE: [ArtifactsManager.js/LIFECYCLE] artifactsManager.onBeforeEach({ title: 'should have welcome screen',
  | fullName: 'Example should have welcome screen',
  | status: 'running' })
  | detox[12084] TRACE: [AsyncWebSocket.js/WEBSOCKET_SEND] {"type":"reactNativeReload","params":{},"messageId":-1000}
  | detox[12084] TRACE: [DetoxServer.js/MESSAGE] role=tester action=reactNativeReload (sessionId=aad9a32a-e684-bd2c-0680-5cecfdf181c0)
  | detox[12084] TRACE: [AsyncWebSocket.js/WEBSOCKET_SEND] {"type":"invoke","params":{"target":{"type":"Invocation","value":{"target":{"type":"EarlGrey","value":"instance"},"method":"detox_selectElementWithMatcher:","args":[{"type":"Invocation","value":{"target":{"type":"Class","value":"GREYMatchers"},"method":"matcherForAccessibilityID:","args":[{"type":"NSString","value":"welcome"}]}}]}},"method":"assertWithMatcher:","args":[{"type":"Invocation","value":{"target":{"type":"Class","value":"GREYMatchers"},"method":"matcherForSufficientlyVisible","args":[]}}]},"messageId":2}
  | detox[12084] TRACE: [DetoxServer.js/MESSAGE] role=tester action=invoke (sessionId=aad9a32a-e684-bd2c-0680-5cecfdf181c0)
  | Example: should have welcome screen [FAIL]
  |  
  | detox[12084] TRACE: [Detox.js/DETOX_AFTER_EACH] failed test: "Example should have welcome screen"
  | detox[12084] TRACE: [ArtifactsManager.js/LIFECYCLE] artifactsManager.onAfterEach({ title: 'should have welcome screen',
  | fullName: 'Example should have welcome screen',
  | status: 'failed',
  | timedOut: true })
  | detox[12084] WARN:  [Client.js/PENDING_REQUESTS] App has not responded to the network requests below:
  | (id = 2) invoke: {"target":{"type":"Invocation","value":{"target":{"type":"EarlGrey","value":"instance"},"method":"detox_selectElementWithMatcher:","args":[{"type":"Invocation","value":{"target":{"type":"Class","value":"GREYMatchers"},"method":"matcherForAccessibilityID:","args":[{"type":"NSString","value":"welcome"}]}}]}},"method":"assertWithMatcher:","args":[{"type":"Invocation","value":{"target":{"type":"Class","value":"GREYMatchers"},"method":"matcherForSufficientlyVisible","args":[]}}]}
  | (id = -1000) reactNativeReload: {}
  |  
  | That might be the reason why the test "Example should have welcome screen" has timed out.
  |  
  | detox[12084] TRACE: [ArtifactsManager.js/LIFECYCLE] artifactsManager.onAfterAll()
  | detox[12084] TRACE: [AsyncWebSocket.js/WEBSOCKET_SEND] {"type":"cleanup","params":{"stopRunner":true},"messageId":-49642}
  | detox[12084] TRACE: [DetoxServer.js/MESSAGE] role=tester action=cleanup (sessionId=aad9a32a-e684-bd2c-0680-5cecfdf181c0)
  | detox[12084] TRACE: [DetoxServer.js/MESSAGE] role=testee action=cleanupDone (sessionId=aad9a32a-e684-bd2c-0680-5cecfdf181c0)
  | detox[12084] TRACE: [AsyncWebSocket.js/WEBSOCKET_MESSAGE] {"type":"cleanupDone","messageId":-49642,"params":{}}
  |  
  | detox[12084] DEBUG: [DetoxServer.js/DISCONNECT] role=tester, sessionId=aad9a32a-e684-bd2c-0680-5cecfdf181c0
  | detox[12084] DEBUG: [DetoxServer.js/DISCONNECT] role=testee, sessionId=aad9a32a-e684-bd2c-0680-5cecfdf181c0
  | detox[12084] DEBUG: [DetoxServer.js/CANNOT_FORWARD] role=tester not connected, cannot fw action (sessionId=aad9a32a-e684-bd2c-0680-5cecfdf181c0)
  | detox[12084] DEBUG: [DetoxServer.js/WS_CLOSE] Detox server connections terminated gracefully
  | detox[12084] DEBUG: [exec.js/EXEC_CMD, #10] /usr/bin/xcrun simctl shutdown 08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D
  | detox[12084] DEBUG: [exec.js/EXEC_TRY, #10] Shutting down 08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D...
  | detox[12084] TRACE: [exec.js/EXEC_SUCCESS, #10]
  | detox[12084] DEBUG: [exec.js/EXEC_SUCCESS, #10] 08EA90D0-FE4B-46D4-98CA-DCD60E93AD3D shut down
  | FAIL e2e/firstTest.spec.js (292.899s)
  | Example
  | ✕ should have welcome screen (240026ms)
  |  
  | ● Example › should have welcome screen
  |  
  | Timeout - Async callback was not invoked within the 120000ms timeout specified by jest.setTimeout.Error: Timeout - Async callback was not invoked within the 120000ms timeout specified by jest.setTimeout.
  |  
  | at mapper (../node_modules/jest-jasmine2/build/queueRunner.js:25:45)
  |  
  | ● Example › should have welcome screen
  |  
  | : Timeout - Async callback was not invoked within the 120000ms timeout specified by jest.setTimeout.Timeout - Async callback was not invoked within the 120000ms timeout specified by jest.setTimeout.Error:
  |  
  | 4 \|   });
  | 5 \|
  | > 6 \|   it('should have welcome screen', async () => {
  | \|   ^
  | 7 \|     await expect(element(by.id('welcome'))).toBeVisible();
  | 8 \|   });
  | 9 \| });
  |  
  | at new Spec (../node_modules/jest-jasmine2/build/jasmine/Spec.js:116:22)
  | at Suite.it (firstTest.spec.js:6:3)
  | at Object.describe (firstTest.spec.js:1:1)
  |  
  | mand failed: node_modules/.bin/jest --config=e2e/config.json --maxWorkers=1 '--testNamePattern=^((?!:android:).)*$' "e2e"
  |  
  | exit code: 1 duration: 294s
  | export SEMAPHORE_JOB_RESULT=failed
```

### semaphore.yml
```
version: v1.0
name: Semaphore React Native Pipeline
agent:
  machine:
    type: a1-standard-4
    os_image: macos-mojave-xcode10
blocks:
  - name: Install dependencies
    task:
      jobs:
        - name: npm install and cache
          commands:
            - xcrun simctl list
            - checkout
            - cache restore node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum package-lock.json),node-modules-$SEMAPHORE_GIT_BRANCH,node-modules-master
            - npm install
            - cache store node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum package-lock.json) node_modules
  - name: Run tests
    task:
      prologue:
        commands:
          - checkout
          - cache restore node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum package-lock.json),node-modules-$SEMAPHORE_GIT_BRANCH,node-modules-master
      jobs:
        - name: e2e tests
          commands:
            - brew tap wix/brew
            - brew install applesimutils
            - cache restore node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum package-lock.json),node-modules-$SEMAPHORE_GIT_BRANCH,node-modules-master
            - npm install -g react-native-cli
            - npm install -g detox-cli
            - cache store node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum package-lock.json) node_modules
            - cd ios && pod install && cd ..
            - detox clean-framework-cache && detox build-framework-cache
            - detox build --configuration ios.sim.debug
            - detox test --configuration ios.sim.debug --cleanup --loglevel trace
```

**Comment by LeoNatan** [comment]
Hi,

I'm not sure what is expected of us here. There are docs explaining how to debug sync issues. There have been countless issues where we've stated how to debug the sync issues. There Stack Overflow.

If you find an actual issue in Detox, please open a new issue. Closing for now.
